src/core/AutomationClip.cpp

```diff
@@ -422,6 +422,37 @@ void AutomationClip::resetNodes(const int tick0, const int tick1)
 
 
 
+/**
+ * @brief Resets the tangents from the nodes between the given ticks
+ * @param Int first tick of the range
+ * @param Int second tick of the range
+ */
+void AutomationClip::resetTangents(const int tick0, const int tick1)
+{
+	if (tick0 == tick1)
+	{
+		auto it = m_timeMap.find(TimePos(tick0));
+		if (it != m_timeMap.end())
+		{
+			it.value().setLockedTangents(false);
+			generateTangents(it, 1);
+		}
+		return;
+	}
+
+	TimePos start = TimePos(qMin(tick0, tick1));
+	TimePos end = TimePos(qMax(tick0, tick1));
```

Comment:
```suggestion
	TimePos start = TimePos(std::min(tick0, tick1));
	TimePos end = TimePos(std::max(tick0, tick1));
```
We should keep `qMin`, `qMax`, and other such Qt functions out of the core. You might need to include `<algorithm>` if it isn't already.